src/App.tsx

```diff
@@ -46,14 +59,31 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const App = () => {
+const defaultSchemaDecorators: PropertiesSchemasDecorator[] = [
+  labelDecorator,
+  multilineStringOptionDecorator,
+  labelUIElementDecorator,
+  ruleDecorator,
+];
+const defaultSchemaProviders: PropertiesSchemasProvider[] = [
+  propertiesSchemaProvider,
+];
+interface AppProps {
+  schemaProviders?: PropertiesSchemasProvider[];
+  schemaDecorators?: PropertiesSchemasDecorator[];
+}
+const App: React.FC<AppProps> = ({ schemaProviders, schemaDecorators }) => {
+  const decorators = schemaDecorators ?? defaultSchemaDecorators;
```

Comment:
I'm not sure it is the best thing to define and use `defaultSchemaDecorators`, maybe we should pass the props rather using the index.tsx? @sdirix  what is your take on this?

Reply:
Not sure if we should offer defaults or not. How will the other services be configured?

Reply:
When we expect all instances to set `schemaDecorators` anyway then I would go with Eugen's suggestion. If we expect that they will not be customized that often, then I'm completely fine with the optional approach. However I would use regular Javascript defaults, i.e.
```typescript
interface AppProps {
  schemaProviders: PropertySchemasProvider[];
  schemaDecorators?: PropertySchemasDecorator[];
}
const App: React.FC<AppProps> = ({
  schemaProviders,
  schemaDecorators = defaultSchemaDecorators,
}) => {
```
In the end I think both approaches are fine here.